core/templates/components/code-mirror/codemirror-mergeview.component.spec.ts

```diff
@@ -99,17 +103,18 @@ describe('Oppia CodeMirror Component', () => {
     );
   });
 
-  it('should not allow undefined for left or right pane', () => {
+  it('should allow for a value of left or right pane', () => {
     const editSetValueSpy = jasmine.createSpy('editSetValueSpy');
     const rightOrgSetValueSpy = jasmine.createSpy('rightOrgSetValueSpy');
-    component.codeMirrorInstance = {
+    spyOnProperty(
+      component, 'getCodeMirrorInstance', 'get').and.returnValue({
       editor: () => {
-        return {setValue: editSetValueSpy} as unknown as CodeMirror.Editor;
+        return { setValue: editSetValueSpy };
       },
       rightOriginal: () => {
-        return {setValue: rightOrgSetValueSpy} as unknown as CodeMirror.Editor;
+        return { setValue: rightOrgSetValueSpy };
       }
-    } as unknown as CodeMirror.MergeView.MergeViewEditor;
```

Comment:
@srijanreddy98 I made that method for this test and the one above it to remove the `as unknown as X` pattern.How can spying on (window.CodeMirror, 'MergeView') help here?